DatadogSessionReplay/Tests/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/ViewTreeSnapshotBuilderTests.swift

```diff
@@ -31,33 +32,6 @@ class ViewTreeSnapshotBuilderTests: XCTestCase {
         XCTAssertEqual(queryContext.recorder, randomRecorderContext)
     }
 
-    func testItConfiguresTextObfuscatorsAccordinglyToCurrentPrivacyMode() throws {
-        // Given
```

Comment:
This test no longer makes sense - we don't configure obfuscators in `VTSBuilder` anymore.